converters/google/resources/resource_converters.go

```diff
@@ -250,6 +250,7 @@ func ResourceConverters() map[string][]ResourceConverter {
 		"google_project_iam_member":           {resourceConverterProjectIamMember()},
 		"google_project_iam_custom_role":      {resourceConverterProjectIAMCustomRole()},
 		"google_organization_iam_custom_role": {resourceConverterOrganizationIAMCustomRole()},
+		"google_org_policy_policy":            {resourceConverterCustomOrgPolicy()},
```

Comment:
It would be preferable to match the name of the Terraform resource.

```suggestion
		"google_org_policy_policy":            {resourceConverterOrgPolicyPolicy()},
```